config/config_helpers.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 var INTERPOLATION_SYNTAX_REGEX = regexp.MustCompile("\\$\\{.*?\\}")
-var HELPER_FUNCTION_SYNTAX_REGEX = regexp.MustCompile("\\$\\{(.*?)\\(\\)\\}")
+var HELPER_FUNCTION_SYNTAX_REGEX = regexp.MustCompile(`\$\{(.*?)\((.*?)\)\}`)
+var HELPER_FUNCTION_GET_ENV_PARAMETERS_SYNTAX_REGEX = regexp.MustCompile(`\"(?P<ENV>[a-zA-Z_]+[a-zA-Z0-9_]+?)\"\,\s*\"(?P<default>.*?)\"`)
```

Comment:
Nice use of named regular expressions! 

However, a few questions:

First, I notice you're escaping double quotes. Since your regex string uses backticks, is that really necessary?

Second, I'm looking at this expression:

> `[a-zA-Z_]+[a-zA-Z0-9_]+?`

Is there a benefit to being strict with the types of characters allowed? For example, I'd have to go dig through the rules for Unix/Linux/OS X/Windows to verify if environment variables are indeed limited to alphanumeric/underscore. I'm guessing that's correct, but it seems like that could be simplified to a simple `.+?` without losing much.

Third, I noticed you're only handling whitespace here:

> "\,\s*

I think you'd need a `\s*` before the comma too. Moreover, people might use whitespace before or after the parens: e.g. `get_env(   "foo"   ,  "bar"    )`.

And finally, fourth, `ENV` and `default` should probably use consistent case: either both upper case or both lower case.